edifact-edi/2.1/modules/ROOT/pages/edifact-edi-connector-reference.adoc

```diff
@@ -73,6 +73,7 @@
 | Initial message reference | String |  The initial Message Reference Number used for outgoing messages. |  1 |
 | Request acknowledgments | Boolean |  Request acknowledgments for send interchanges using Acknowledgment Request field (UNB.9). If true, CONTRL acknowledgments are requested in all sent interchanges. If false, only receipt confirmations are requested. |  false |
 | Test indicator | String |  Test indicator digit to be used on send interchanges. By default, interchanges are not sent as tests. |  |
+| Use Supplied Values | Boolean | Whether to use supplied values for control segment information such as control numbers. The default behavior is to always generate control numbers when writing. | false |
```

Comment:
Is "control segment information" a type of info? If so, hyphenate:
```suggestion
| Use Supplied Values | Boolean | Whether to use supplied values for control-segment information such as control numbers. The default behavior is to always generate control numbers when writing. | false |
```
Should the first column value use sentence-style caps. like its peers: Use supplied values?

Please clarify: if the default value is `false`, then the default behavior is to generate NEW values rather than using the supplied values, right? Consider rewording to clarify.

Reply:
No on the first one, reworded the second comment.